### PR TITLE
Add Japanese&Chinese

### PR DESCRIPTION
**Pull request by EldersJavas**
reference
REC-xml-20081126
RFC1766
EBU tech 3264

### CODE REVIEW

File: language.go [hunk 1]
```diff
@@ -5,4 +5,6 @@ const (
 	LanguageEnglish   = "english"
 	LanguageFrench    = "french"
 	LanguageNorwegian = "norwegian"
+	LanguageChinese = "chinese"
```
Comment (asticode) [inline language.go:8]: Could you order constants alphabetically?

File: stl.go [hunk 2]
```diff
@@ -108,8 +108,10 @@ const (
 
 // STL country codes
 const (
-	stlCountryCodeFrance = "FRA"
-	stlCountryCodeNorway = "NOR"
+	stlCountryCodeFrance  = "FRA"
+	stlCountryCodeNorway  = "NOR"
+	stlCountryCodeChinese = "CHN"
```
Comment (asticode) [inline stl.go:113]: Could you order constants alphabetically?

File: stl.go [hunk 3]
```diff
@@ -145,15 +147,19 @@ const (
 	stlLanguageCodeEnglish   = "09"
 	stlLanguageCodeFrench    = "0F"
 	stlLanguageCodeNorwegian = "1E"
+	stlLanguageCodeChinese   = "75"
```
Comment (asticode) [inline stl.go:150]: Could you order constants alphabetically?

File: stl.go [hunk 4]
```diff
@@ -145,15 +147,19 @@ const (
...
+	stllanguageCodeJapanese  = "69"
 )
 
 // STL language mapping
 var stlLanguageMapping = astikit.NewBiMap().
 	Set(stlLanguageCodeEnglish, LanguageEnglish).
 	Set(stlLanguageCodeFrench, LanguageFrench).
-	Set(stlLanguageCodeNorwegian, LanguageNorwegian)
+	Set(stlLanguageCodeNorwegian, LanguageNorwegian).
```
Comment (asticode) [inline stl.go:158]: Could you order lines alphabetically?

File: ttml.go [hunk 5]
```diff
@@ -22,13 +22,18 @@ const (
 	ttmlLanguageEnglish   = "en"
 	ttmlLanguageFrench    = "fr"
 	ttmlLanguageNorwegian = "no"
+	ttmlLanguageChinese   = "zh"
```
Comment (asticode) [inline ttml.go:25]: Could you order constants alphabetically?

File: ttml.go [hunk 6]
```diff
@@ -22,13 +22,18 @@ const (
...
+	ttmlLanguageJapanese  = "ja"
+
 )
 
 // TTML language mapping
 var ttmlLanguageMapping = astikit.NewBiMap().
 	Set(ttmlLanguageEnglish, LanguageEnglish).
 	Set(ttmlLanguageFrench, LanguageFrench).
-	Set(ttmlLanguageNorwegian, LanguageNorwegian)
+	Set(ttmlLanguageNorwegian, LanguageNorwegian).
```
Comment (asticode) [inline ttml.go:34]: Could you order lines alphabetically?

**Review by asticode — CHANGES_REQUESTED** [review]
This is a nice PR.

Just minor changes needed though

**Comment by EldersJavas** [comment]
sloved :)

**Comment by asticode** [comment]
FYI I've created a `v0.16.0` tag

**Pull request merged**